src/main/java/com/gooddata/md/visualization/VisualizationClass.java

```diff
@@ -0,0 +1,102 @@
+/*
+ * Copyright (C) 2018, GoodData(R) Corporation. All rights reserved.
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE.txt file in the root directory of this source tree.
+ */
+
+package com.gooddata.md.visualization;
+
+import com.fasterxml.jackson.annotation.*;
+import com.gooddata.md.AbstractObj;
+import com.gooddata.md.Meta;
+import com.gooddata.md.Queryable;
+import com.gooddata.md.Updatable;
+
+import java.io.Serializable;
+
+import static com.fasterxml.jackson.annotation.JsonTypeInfo.As.WRAPPER_OBJECT;
+import static com.fasterxml.jackson.annotation.JsonTypeInfo.Id.NAME;
+import static org.apache.commons.lang3.Validate.notNull;
+
+@JsonTypeName(VisualizationClass.NAME)
+@JsonTypeInfo(include = WRAPPER_OBJECT, use = NAME)
+@JsonIgnoreProperties(ignoreUnknown = true)
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class VisualizationClass extends AbstractObj implements Queryable, Updatable {
+
+    private static final long serialVersionUID = -72785788784079208L;
+    static final String NAME = "visualizationClass";
+
+    private Content content;
+
+    public VisualizationClass(@JsonProperty("content") final Content content, @JsonProperty("meta") final Meta meta ) {
+        super(meta);
+        this.content = notNull(content);
+    }
+
+    public Content getContent()  {
+        return content;
+    }
+
+    @JsonIgnore
+    public VisualizationType getVisualizationType() {
+        String uriParts[] = getContent().getUrl().split(":");
+        String derivedType = uriParts[uriParts.length-1];
```

Comment:
Can we be sure that the array is not empty? Maybe add a check?

Reply:
Also, can we be sure that `getUrl()` does not return null?

Reply:
With array not 100%, adding check. getUrl should never return null because of notNull check in constructor